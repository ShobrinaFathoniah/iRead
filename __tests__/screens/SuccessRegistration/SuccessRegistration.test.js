import * as React from 'react';
import SuccessRegistration from '../../../src/screens/SuccessRegister';
import {create} from 'react-test-renderer';
import ContainerTesting from '../../../src/helpers/reduxTesting';

test('SuccessRegistration Snapshoot', () => {
  const snap = create(ContainerTesting(<SuccessRegistration />));
  expect(snap).toMatchSnapshot();
});
