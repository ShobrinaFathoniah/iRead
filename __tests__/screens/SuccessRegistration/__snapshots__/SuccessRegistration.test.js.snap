// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessRegistration Snapshoot 1`] = `
<View>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "Courgette-Regular",
        },
        Object {
          "alignSelf": "center",
          "color": "#272b5a",
          "fontSize": 44,
          "letterSpacing": 1.5714285714285714,
          "marginTop": 94.28571428571428,
        },
      ]
    }
  >
    Registration Completed!
  </Text>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      Array [
        Object {
          "color": "#8180b7",
          "fontSize": 150,
        },
        Object {
          "alignSelf": "center",
          "marginTop": 157.14285714285714,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï„¬
  </Text>
  <Text
    ellipsizeMode="tail"
    numberOfLines={0}
    style={
      Array [
        Object {
          "fontFamily": "LibreBaskerville-Regular",
        },
        Object {
          "color": "#535487",
          "fontSize": 29.857142857142858,
          "marginTop": 78.57142857142857,
          "padding": 7.857142857142857,
          "textAlign": "center",
        },
      ]
    }
  >
    We sent email verification to your email, Please Check It!
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#53548799",
        "borderRadius": 4.714285714285714,
        "margin": 15.714285714285714,
        "marginTop": 94.28571428571428,
        "opacity": 1,
        "padding": 15.714285714285714,
        "width": 408.57142857142856,
      }
    }
  >
    <Text
      ellipsizeMode="tail"
      numberOfLines={0}
      style={
        Array [
          Object {
            "fontFamily": "LibreBaskerville-Regular",
          },
          Object {
            "alignSelf": "center",
            "color": "#FFFFFF",
            "fontSize": 22,
          },
        ]
      }
    >
      Back to Login
    </Text>
  </View>
</View>
`;
